### PR TITLE
N16: FBSim release and tutorial

### PR DESCRIPTION
**Pull request by IanTayler**
Part of #365 

First contribution to the newsletter, so let me know of any potential improvements to the text or PR.

### CODE REVIEW

File: content/posts/newsletter-016/index.md [hunk 1]
```diff
@@ -63,6 +63,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+_Discussions:
+[/r/rust_gamedev](https://reddit.com/r/rust_gamedev/comments/jz4x1f/fbsim/)_
+
+[FBSim]: https://github.com/IanTayler/fbsim
+[github repo]: https://github.com/IanTayler/fbsim
+[Ian Tayler]: https://iantayler.com
+[tutorial]: https://iantayler.com/2020/11/22/fbsim-football-playing-ai-agents-in-rust/
+
+
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-016/index.md:87]:
```suggestion

```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks good, just one formatting thing that's tripping up the CI.

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:fire:

**Comment by ozkriff** [comment]
Thanks!

**Pull request merged**